src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Collect.txt

```diff
@@ -54,4 +54,29 @@ Errors: Error 8-11: Name isn't valid. 'Foo' isn't recognized.|Error 0-15: The fu
 Errors: Error 8-11: Name isn't valid. 'Foo' isn't recognized.|Error 12-15: Name isn't valid. 'Bar' isn't recognized.|Error 0-16: The function 'Collect' has some invalid arguments.
 
 >> Collect(1/0,Foo)
-Errors: Error 12-15: Name isn't valid. 'Foo' isn't recognized.
\ No newline at end of file
+Errors: Error 12-15: Name isn't valid. 'Foo' isn't recognized.
+
+>> Collect(Error({Kind:ErrorKind.Custom}), r2)
+Errors: Error 8-38: Invalid argument type (ObjNull). Expecting a Table value instead.|Error 0-43: The function 'Collect' has some invalid arguments.
+
+// !JYL! (ObjNull)???
+>> Collect(Error({Kind:ErrorKind.Custom}), Error({Kind:ErrorKind.Custom}))
+Errors: Error 8-38: Invalid argument type (ObjNull). Expecting a Table value instead.|Error 0-71: The function 'Collect' has some invalid arguments.
+
+// !JYL! (ObjNull)???
+>> Collect(Blank(), r2)
+Errors: Error 8-15: Invalid argument type (ObjNull). Expecting a Table value instead.|Error 0-20: The function 'Collect' has some invalid arguments.
+
+// !JYL! (ObjNull)???
+>> Collect(Blank(), Blank())
+Errors: Error 8-15: Invalid argument type (ObjNull). Expecting a Table value instead.|Error 0-25: The function 'Collect' has some invalid arguments.
+
+>> Collect("", r2)
+Errors: Error 8-10: Invalid argument type (Text). Expecting a Table value instead.|Error 0-15: The function 'Collect' has some invalid arguments.
+
+>> Collect("", "")
+Errors: Error 8-10: Invalid argument type (Text). Expecting a Table value instead.|Error 12-14: Invalid argument type (Text). Expecting a Record value instead.|Error 0-15: The function 'Collect' has some invalid arguments.
+
+// !JYL! (_Min)???
```

Comment:
This seems very odd! 
What does canvas say?

Reply:
This has been resolved.